apps/aechannel/src/aesc_session_noise.erl

```diff
@@ -135,152 +167,24 @@ patterns() ->
 
 record_fields(st) -> record_info(fields, st);
 record_fields(_ ) -> no.
-%% ==================================================================
-
-%% Connection establishment
-
-connect(Host, Port, Opts) ->
-    start_link(#{fsm => self(), op => {connect, Host, Port, Opts}}).
```

Comment:
I changed the connect call to start the process through the supervisor. @uwiger Why wasn't that the case before?